packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/include/video_player_avfoundation/FVPNativeVideoViewFactory.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Platform views are only supported on iOS as of now. Ifdef is used to avoid compilation errors.
+
+#import <TargetConditionals.h>
```

Comment:
This file should import `Foundation` given the types it's using, at which point you won't need this.